### PR TITLE
CortexDataLake/update the name and logo

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:  https://jira-hq.paloaltonetworks.local/browse/CIAC-5796

## Description
Update the logo and name of the CortexDataLake integration.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 1]
```diff
@@ -130,11 +130,11 @@ configuration:
   name: insecure
   required: false
   type: 8
-description: Palo Alto Networks Cortex Data Lake provides cloud-based, centralized
+description: Palo Alto Networks Cortex XSOAR Data Lake provides cloud-based, centralized
   log storage and aggregation for your organization on premise, virtual (private cloud
   and public cloud) firewalls, for Prisma Access, and for cloud-delivered services
   such as Cortex XDR.
-display: Cortex Data Lake
+display: Cortex XSOAR Data Lake
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:137]:
```suggestion
display: Cortex Data Lake XSOAR Connector
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 2]
```diff
@@ -130,11 +130,11 @@ configuration:
   name: insecure
   required: false
   type: 8
-description: Palo Alto Networks Cortex Data Lake provides cloud-based, centralized
+description: Palo Alto Networks Cortex XSOAR Data Lake provides cloud-based, centralized
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:133]:
```suggestion
description: Palo Alto Networks Cortex Data Lake XSOAR Connector provides cloud-based, centralized
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 3]
```diff
@@ -401,7 +401,7 @@ script:
       description: Severity associated with the event.
       type: String
     - contextPath: CDL.Logging.Threat.LogTime
-      description: Time the log was received in Cortex Data Lake.
+      description: Time the log was received in Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:404]:
```suggestion
      description: Time the log was received in Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 4]
```diff
@@ -487,7 +487,7 @@ script:
         (NAT) for the logged traffic. If it is, this value is 1, otherwise it is 0.
       type: Boolean
     - contextPath: CDL.Logging.Traffic.LogTime
-      description: Time the log was received in Cortex Data Lake.
+      description: Time the log was received in Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:490]:
```suggestion
      description: Time the log was received in Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 5]
```diff
@@ -748,7 +748,7 @@ script:
       description: Severity associated with the event.
       type: String
     - contextPath: CDL.Logging.Threat.LogTime
-      description: Time the log was received in Cortex Data Lake.
+      description: Time the log was received in Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:751]:
```suggestion
      description: Time the log was received in Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 6]
```diff
@@ -1057,7 +1057,7 @@ script:
         (NAT) for the logged traffic. If it is, this value is 1, otherwise it is 0.
       type: Boolean
     - contextPath: CDL.Logging.Traffic.LogTime
-      description: Time the log was received in Cortex Data Lake.
+      description: Time the log was received in Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:1060]:
```suggestion
      description: Time the log was received in Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 7]
```diff
@@ -1571,7 +1571,7 @@ script:
       description: Severity associated with the event.
       type: String
     - contextPath: CDL.Logging.Threat.LogTime
-      description: Time the log was received in Cortex Data Lake.
+      description: Time the log was received in Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:1574]:
```suggestion
      description: Time the log was received in Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 8]
```diff
@@ -1936,7 +1936,7 @@ script:
       description: Identifies the log subtype.
       type: String
     - contextPath: CDL.Logging.URL.LogTime
-      description: The time the log was received in Cortex Data Lake.
+      description: The time the log was received in Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:1939]:
```suggestion
      description: The time the log was received in Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 9]
```diff
@@ -2470,10 +2470,10 @@ script:
       type: Number
     - contextPath: CDL.Logging.File.IsDupLog
       description: Indicates whether this log data is available in multiple locations,
-        such as from Cortex Data Lake as well as from an on-premise log collector.
+        such as from Cortex XSOAR Data Lake as well as from an on-premise log collector.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:2473]:
```suggestion
        such as from Cortex Data Lake XSOAR Connector as well as from an on-premise log collector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 10]
```diff
@@ -2470,10 +2470,10 @@ script:
...
       type: Boolean
     - contextPath: CDL.Logging.File.LogTime
-      description: Time the log was received in Cortex Data Lake.
+      description: Time the log was received in Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:2476]:
```suggestion
      description: Time the log was received in Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 11]
```diff
@@ -2550,7 +2550,7 @@ script:
       description: IP protocol associated with the session.
       type: String
     - contextPath: CDL.Logging.File.CustomerID
-      description: The ID that uniquely identifies the Cortex Data Lake instance which
+      description: The ID that uniquely identifies the Cortex XSOAR Data Lake instance which
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:2553]:
```suggestion
      description: The ID that uniquely identifies the Cortex Data Lake XSOAR Connector instance which
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake_description.md [hunk 12]
```diff
@@ -1,14 +1,14 @@
 ## Overview
 ---
 
-Palo Alto Networks Cortex Data Lake provides cloud-based, centralized log storage and aggregation for your on premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR
-This integration was integrated and tested with version 2 of Cortex Data Lake
+Palo Alto Networks Cortex XSOAR Data Lake provides cloud-based, centralized log storage and aggregation for your on premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake_description.md:4]:
```suggestion
Palo Alto Networks Cortex Data Lake XSOAR Connector provides cloud-based, centralized log storage and aggregation for your on premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake_description.md [hunk 13]
```diff
@@ -1,14 +1,14 @@
...
+This integration was integrated and tested with version 2 of Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake_description.md:5]:
```suggestion
This integration was integrated and tested with version 2 of Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake_description.md [hunk 14]
```diff
@@ -1,14 +1,14 @@
...
 
 
 
 ---
 
-## Configure Cortex Data Lake on XSOAR
+## Configure Cortex XSOAR Data Lake on XSOAR
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake_description.md:11]:
```suggestion
## Configure Cortex Data Lake XSOAR Connector on Cortex XSOAR 
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 15]
```diff
@@ -1,12 +1,12 @@
 ## Overview
 ---
 
-Palo Alto Networks Cortex Data Lake provides cloud-based, centralized log storage and aggregation for your on premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR
-This integration was integrated and tested with version 2 of Cortex Data Lake
+Palo Alto Networks Cortex XSOAR Data Lake provides cloud-based, centralized log storage and aggregation for your on premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:4]:
```suggestion
Palo Alto Networks Cortex Data Lake XSOAR Connector provides cloud-based, centralized log storage and aggregation for your on premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 16]
```diff
@@ -1,12 +1,12 @@
...
+This integration was integrated and tested with version 2 of Cortex XSOAR Data Lake
 
 ---
 
-## Configure Cortex Data Lake on Cortex XSOAR
+## Configure Cortex XSOAR Data Lake on Cortex XSOAR
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:9]:
```suggestion
## Configure Cortex Data Lake XSOAR Connector on Cortex XSOAR
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 17]
```diff
@@ -19,7 +19,7 @@ The License ID will be used in Step 4.
 3. In Palo Alto Networks Cortex XSOAR, go to __Settings__ > __ABOUT__ > __License__ and get the value in the license field Customer name. The Customer name will be used in Step 4.
 4. In the Palo Alto Networks HUB, enter the License ID and the Customer name in the screen obtained in Step 1. The License ID and Customer name were obtained in Steps 2 and 3. Click **Start Authorization Process** to get the Authentication Token, Registration ID, and Encryption Key - these three fields will be used in the Palo Alto Networks Cortex v2 integration instance in Step 7 below.
 5. In Palo Alto Networks Cortex XSOAR, navigate to __Settings__ > __Integrations__ > __Servers & Services__.
-6. Search for Cortex Data Lake.
+6. Search for Cortex XSOAR Data Lake.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:22]:
```suggestion
6. Search for Cortex Data Lake XSOAR Connector.
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 18]
```diff
@@ -339,7 +339,7 @@ Runs a query on the Cortex logging service, according to preset queries.
 | CDL.Logging.Traffic.App | String | Application associated with the network traffic. | 
 | CDL.Logging.Traffic.Vsys | String | Virtual system associated with the network traffic. | 
 | CDL.Logging.Traffic.IsNat | String | Indicates whether the firewall is performing network address translation (NAT) for the logged traffic. If it is, this value is 1. | 
-| CDL.Logging.Traffic.LogTime | date | Time the log was received in Cortex Data Lake. | 
+| CDL.Logging.Traffic.LogTime | date | Time the log was received in Cortex XSOAR Data Lake. |
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:342]:
```suggestion
| CDL.Logging.Traffic.LogTime | date | Time the log was received in Cortex Data Lake XSOAR Connector. | 
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 19]
```diff
@@ -650,7 +650,7 @@ Searches the Cortex firewall.traffic table. Traffic logs contain entries for the
...
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:653]:
```suggestion
| CDL.Logging.Traffic.LogTime | date | Time the log was received in Cortex Data Lake XSOAR Connector. | 
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 20]
```diff
@@ -938,7 +938,7 @@ Searches the URL table
 | CDL.Logging.URL.ThreatCategory | String | Threat category of the detected threat. | 
 | CDL.Logging.URL.ThreatName | String | Threat name of the detected threat. | 
 | CDL.Logging.URL.Subtype | String | Identifies the log subtype. | 
-| CDL.Logging.URL.LogTime | String | Time the log was received in Cortex Data Lake. | 
+| CDL.Logging.URL.LogTime | String | Time the log was received in Cortex XSOAR Data Lake. |
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:941]:
```suggestion
| CDL.Logging.URL.LogTime | String | Time the log was received in Cortex Data Lake XSOAR Connector.| 
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 21]
```diff
@@ -1125,8 +1125,8 @@ Searches the Cortex firewall.file_data table.
 | CDL.Logging.File.ReportID | Number | Identifies the analysis requested from the sandbox \(cloud or appliance\). | 
 | CDL.Logging.File.DestinationPort | Number | Network traffic's destination port. If this value is 0, then the app is using
 its standard port. | 
-| CDL.Logging.File.IsDupLog | Boolean | Indicates whether this log data is available in multiple locations, such as from Cortex Data Lake as well as from an on-premise log collector. | 
-| CDL.Logging.File.LogTime | Date | Time the log was received in Cortex Data Lake. | 
+| CDL.Logging.File.IsDupLog | Boolean | Indicates whether this log data is available in multiple locations, such as from Cortex XSOAR Data Lake as well as from an on-premise log collector. |
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:1128]:
```suggestion
| CDL.Logging.File.IsDupLog | Boolean | Indicates whether this log data is available in multiple locations, such as from Cortex Data Lake XSOAR Connector as well as from an on-premise log collector. | 
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 22]
```diff
@@ -1125,8 +1125,8 @@ Searches the Cortex firewall.file_data table.
 | CDL.Logging.File.ReportID | Number | Identifies the analysis requested from the sandbox \(cloud or appliance\). | 
 | CDL.Logging.File.DestinationPort | Number | Network traffic's destination port. If this value is 0, then the app is using
 its standard port. | 
-| CDL.Logging.File.IsDupLog | Boolean | Indicates whether this log data is available in multiple locations, such as from Cortex Data Lake as well as from an on-premise log collector. | 
-| CDL.Logging.File.LogTime | Date | Time the log was received in Cortex Data Lake. | 
+| CDL.Logging.File.IsDupLog | Boolean | Indicates whether this log data is available in multiple locations, such as from Cortex XSOAR Data Lake as well as from an on-premise log collector. | 
+| CDL.Logging.File.LogTime | Date | Time the log was received in Cortex XSOAR Data Lake. |
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:1129]:
```suggestion
| CDL.Logging.File.LogTime | Date | Time the log was received in Cortex Data Lake XSOAR Connector. | 
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 23]
```diff
@@ -1298,4 +1298,4 @@ against. That is, log types must be fully qualified and the instance ID is a par
 `<instanceID>.firewall.traffic`
 However in this integration the instance ID is added automatically to the query so the name `firewall.traffic` is a valid table name
 * The SQL syntax supported for queries is `csql`
-* The provided authentication items ([configuration step 4](#configure-cortex-data-lake-on-cortex-xsoar)) can only be used once for each Cortex Data Lake tenant (but can be shared for different Cortex XSOAR instances). Trying to re-generate those items will revoke any previously generated set of authentication items.
\ No newline at end of file
+* The provided authentication items ([configuration step 4](#configure-cortex-data-lake-on-cortex-xsoar)) can only be used once for each Cortex XSOAR Data Lake tenant (but can be shared for different Cortex XSOAR instances). Trying to re-generate those items will revoke any previously generated set of authentication items.
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:1301]:
```suggestion
* The provided authentication items ([configuration step 4](#configure-cortex-data-lake-on-cortex-xsoar)) can only be used once for each Cortex Data Lake XSOAR Connector tenant (but can be shared for different Cortex XSOAR instances). Trying to re-generate those items will revoke any previously generated set of authentication items.
```

File: Packs/CortexDataLake/ReleaseNotes/1_4_3.md [hunk 24]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Cortex XSOAR Data Lake Integration
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/ReleaseNotes/1_4_3.md:3]:
```suggestion
##### Cortex Data Lake XSOAR Connector Integration
```

File: Packs/CortexDataLake/pack_metadata.json [hunk 25]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "Cortex Data Lake by Palo Alto Networks",
-    "description": "Palo Alto Networks Cortex Data Lake provides cloud-based, centralized log storage and aggregation for your on-premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR",
+    "description": "Palo Alto Networks Cortex XSOAR Data Lake provides cloud-based, centralized log storage and aggregation for your on-premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR",
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/pack_metadata.json:3]:
```suggestion
    "description": "Palo Alto Networks Cortex Data Lake XSOAR Connector provides cloud-based, centralized log storage and aggregation for your on-premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR",
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py [hunk 26]
```diff
@@ -246,7 +246,7 @@ def query_loggings(self, query: str) -> Tuple[List[dict], list]:
             except AttributeError:
                 error_message = query_result
 
-            raise DemistoException(f'Error in query to Cortex Data Lake [{status_code}] - {error_message}')
+            raise DemistoException(f'Error in query to Cortex XSOAR Data Lake [{status_code}] - {error_message}')
```
Comment (ShirleyDenkberg) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py:249]:
```suggestion
            raise DemistoException(f'Error in query to Cortex Data Lake XSOAR Connector [{status_code}] - {error_message}')
```

**Comment by ShirleyDenkberg** [comment]
@gal-forer @bakatzir As agreed, I changed the name of the integration to Cortex Data Lake XSOAR Connector.
Doc review completed.

**Review by bziser — APPROVED** [review]
looks great

**Pull request merged**